### ISSUE TITLE
A link prediction regularization objective in node classification tasks

### ISSUE THREAD

**Issue by cjissmart** [HazyResearch/hgcn#15]
Thanks for releasing the detailed code!
However, I am confused about the sentence in paper. According to my understanding, the meaning of the sentence is that a link prediction regularization is added to the objective function (cross entropy) for node classification tasks. But I don't find the corresponding objective function. For node classification tasks, the loss function is still the cross entropy.
![image](https://user-images.githubusercontent.com/37531166/88122878-3fc08480-cbfc-11ea-82bd-8647f7247693.png)
Or the link prediction regularization is completed in link prediction tasks. And node classification is the downstream of link prediction tasks. When I modify my code in the way, the accuracy of Cora is very low. The accuracy is OK if I comment the part.
![image](https://user-images.githubusercontent.com/37531166/88124075-cece9c00-cbfe-11ea-83d1-a422e346a86d.png)
By the way, the model I use is HGAT.

**Comment by ines-chami** [comment]
Hi, please try using a regularization coefficient to the link prediction loss term before summing (e.g. 0.1, 0.01...). Another option is to pretrain embeddings for link prediction, and then reload pretrained embeddings for node classification with the option --pretrained-embeddings (see the readme file for examples).
